app/routes/schema.py

```diff
@@ -18,11 +15,5 @@ def get_schema_json_from_name(schema_name):
 
 @schema_blueprint.route("/schemas", methods=["GET"])
 def list_schemas():
-    schema_path = "data/en"
-    schema_names = [
-        f.replace(".json", "")
-        for f in listdir(schema_path)
-        if isfile(join(schema_path, f)) and f.endswith(".json")
-    ]
-
-    return jsonify(schema_names)
+    schemas = list(SCHEMA_PATH_MAP["en"].keys())
```

Comment:
The map shouldn't be exposed outside of `app.utilities.schema`. We should add a method to `app.utilities.schema` that returns a list of all schemas for a given language code.